deployment/data-streams/changeset/reward-manager/deploy_reward_manager.go

```diff
@@ -0,0 +1,112 @@
+package reward_manager
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/types"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/llo-feeds/generated/reward_manager_v0_5_0"
+)
+
```

Comment:
Functionality to be replaced by https://github.com/smartcontractkit/chainlink/pull/16884

Reply:
Already merged.